designs/virtualization/4.5/Uninstall-cnv/uninstall-cnv.md

```diff
@@ -0,0 +1,27 @@
+---
+parent: Virtualization
+version: 4.5
+---
+
+# Uninstall CNV
+
+Ensure that the user is aware that uninstalling this operator will require to delete all VMs as well. That deletion cannot be reversed and data will be lost.
+
+![Operator details page](img/uninstall-0-0.png)
+
+When `uninstall` is selected the user is presented with a confirmation dialog notifying that:
+- This action requires deletion of all VMs
+- There are [x] VMs found, with an external link to view them
+- A confirmation string box, where the user need to type the operators name.
+
+![Modal migrate](img/uninstall-3-0.jpg)
+
+ After the migration had started, the status of the VM in the list view will be updated to reflect that the migration has already started.
```

Comment:
Is there another image to show this state?

Reply:
My bad, leftovers from the template